test-sender/src/main.rs

```diff
@@ -97,14 +99,14 @@ async fn start_sender(config: TestSenderConfig) {
                     let mut graphcast_message = GraphcastMessage::build(
                         &wallet,
                         topic.clone(),
-                        timestamp,
+                        nonce,
                         "0x7e6528e4ce3055e829a32b5dc4450072bac28bc6".to_string(),
                         radio_payload,
                     )
                     .await
                     .unwrap();
 
-                    graphcast_message.nonce = config.nonce.clone().unwrap().parse::<i64>().unwrap();
+                    graphcast_message.nonce = nonce;
```

Comment:
do you need this? shouldn't it be set to nonce already by L99? It could be an assert check

Reply:
whoops, leftover